numpy/core/fromnumeric.py

```diff
@@ -1154,6 +1165,14 @@ def argmax(a, axis=None, out=None):
     >>> np.argmax(b)  # Only the first occurrence is returned.
     1
 
+    >>> x = np.array([[4,2,3], [1,0,3]])
+    >>> index_array = np.argmax(x, axis=-1)
+    >>> np.take_along_axis(x, np.expand_dims(index_array, axis=-1), axis=-1)  # same as np.max(x, axis=-1, keepdims=True)
+    array([[4],
+           [3]])
+    >>> np.take_along_axis(x, np.expand_dims(index_array, axis=-1), axis=-1).squeeze(axis=-1) # same as np.max(x, axis=-1)
+    array([4, 3])
+
```

Comment:
These lines are quite long. Maybe move the comment to stand alone in the line above the code